kanidmd/idm/src/lib.rs

```diff
@@ -14,6 +14,7 @@
 #![deny(clippy::needless_pass_by_value)]
 #![deny(clippy::trivially_copy_pass_by_ref)]
 
+#[cfg(not(target_family = "windows"))]
```

Comment:
Could this be merged down to the line below as  `#[cfg(all(jemallocator, test, not(target_family = "windows")))]`

Reply:
Done! - I couldn't puzzle out the syntax to work that out and just copypasta'd around. 😇